doc/connectivity/region_extraction.rst

```diff
@@ -27,11 +27,11 @@ Region Extraction for better brain parcellations
 Fetching resting state functional datasets
 ==========================================
 
-We use ADHD resting state functional connectivity datasets of 20 subjects,
+We use resting state functional connectivity datasets of 20 subjects,
 which is already preprocessed and publicly available at
-`<http://fcon_1000.projects.nitrc.org/indi/adhd200/>`_. We use utilities
-:func:`fetch_adhd` implemented in nilearn for automatic fetching of these
-datasets.
+`<https://openneuro.org/datasets/ds000228/versions/1.0.0>`_. We use utilities
```

Comment:
Should we actually give the OSF link here ? That's where the preprocessed data lives. The data available on OpenNeuro is either raw BIDS or smoothed.